build/images/base/build_and_push.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is a very simple script that builds the base image for Antrea and pushes it to
+# the Antrea Dockerhub (https://hub.docker.com/u/antrea). The image is tagged with the OVS version.
+
+set -eo pipefail
+
+function echoerr {
+    >&2 echo "$@"
+}
+
+if [ -z "$OVS_VERSION" ]; then
+    echoerr "The OVS_VERSION env variable must be set to a valid value (e.g. 2.13.0)"
+    exit 1
+fi
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+pushd $THIS_DIR > /dev/null
+
+docker pull ubuntu:18.04
+
+docker build \
+       --cache-from antrea/base:$OVS_VERSION \
+       -t antrea/base:$OVS_VERSION \
+       --build-arg OVS_VERSION=$OVS_VERSION .
```

Comment:
build arg not needed, you don't use it in the dockerfile

Reply:
if you want to use it in the dockerfile replace
```
FROM antrea/openvswitch:2.13.0
```
with
```
ARG OVS_VERSION=2.13.0
FROM antrea/openvswitch:${OVS_VERSION}
```

Reply:
Updated.